functions/__gitnow_check_create_branch.fish

```diff
@@ -0,0 +1,14 @@
+# GitNow — Speed up your Git workflow. 🐠
+# https://github.com/joseluisq/gitnow
+
+function __gitnow_check_create_branch -a xbranch
+    set -l xfound (__gitnow_check_if_branch_exist $xbranch)
+
+    if test $xfound -eq 1
+        echo "Branch `$xbranch` already exists. Nothing to do."
+    else
+        command git stash
+        __gitnow_new_branch_switch "$xbranch"
+        command git stash pop
+    end
+end
```

Comment:
This function should validate the branch name (not empty) and earlier (even before trying to stash).

```fish
$ branch
No local changes to save
fatal: '' is not a valid branch name
No stash entries found.
```